rubin_scheduler/utils/consdb.py

```diff
@@ -110,10 +111,13 @@ class ConsDBVisits(ABC):
         The date for which to query the database.
     url : `str`
         The connection string from the database.
+    num_nights : `int`
+        The number of nights (ending in day_obs) for which to get visits.
```

Comment:
ending in day_obs or "including day_obs" ? 
(just thinking 1 = day_obs itself, not the night before ..)